pwiz_tools/Skyline/Model/Export.cs

```diff
@@ -3046,7 +3046,9 @@ protected override void WriteHeaders(TextWriter writer)
             writer.Write(FieldSeparator);
             writer.Write(@"Unit");   // MS1 Res
             writer.Write(FieldSeparator);
-            writer.Write(GetProductMz(SequenceMassCalc.PersistentMZ(nodeTran.Mz), step).ToString(CultureInfo));
+            // For Agilent we do not call GetProductMz because we want all of the Q3 m/z values to be the same
+            // for all of the optimization step chromatograms
+            writer.Write(SequenceMassCalc.PersistentMZ(nodeTran.Mz).ToString(CultureInfo));

```

Comment:
I would have just replace "step" with 0. What if GetProductMz() somehow gets expanded in the future?